### PR TITLE
add ci-operator release file with fast tag and add periodical builds

### PR DESCRIPTION
**Pull request by rpancham**
Add ci-operator release file with fast tag and added the periodical builds
[246](https://github.com/opendatahub-io/modelmesh-serving/issues/246)

### CODE REVIEW

File: ci-operator/config/opendatahub-io/openvino_model_server/opendatahub-io-openvino_model_server-2023.1-main.yaml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+- as: pr-image-mirror
+  run_if_changed: ^(Dockerfile.redhat)
+  steps:
+    dependencies:
+      SOURCE_IMAGE_REF: model-server-main-cuda-intel-gpu
+    env:
+      IMAGE_REPO: openvino_model_server
+    workflow: opendatahub-io-ci-image-mirror
+- as: fast-image-mirror
+  cron: 0 0 * * 0
```
Comment (Xaenalt) [inline ci-operator/config/opendatahub-io/openvino_model_server/opendatahub-io-openvino_model_server-2023.1-main.yaml:70]: It's looking like the cron job conflicts with the postsubmit job. Let's delete this cron line and add a third task of the form:

```
- as: nightly-image-mirror
  cron: 0 0 * * 0 ?
  steps:
    dependencies:
      SOURCE_IMAGE_REF: caikit-tgis-serving
    env:
      IMAGE_REPO: caikit-tgis-serving
      RELEASE_VERSION: rhods-1.31
    workflow: red-hat-data-services-ci-image-mirror
```

Likely needs an update to the same values here, etc. Would need for both release and main

File: ci-operator/config/opendatahub-io/openvino_model_server/opendatahub-io-openvino_model_server-2023.1-release.yaml [hunk 2]
```diff
@@ -67,6 +67,7 @@ tests:
       IMAGE_REPO: openvino_model_server
     workflow: opendatahub-io-ci-image-mirror
 - as: fast-image-mirror
+  cron: 0 0 * * 0
```
Comment (Xaenalt) [inline ci-operator/config/opendatahub-io/openvino_model_server/opendatahub-io-openvino_model_server-2023.1-release.yaml:70]: Same changes here

**Review by Xaenalt — CHANGES_REQUESTED** [review]

**Comment by rpancham** [comment]
/pj-rehearse pull-ci-opendatahub-io-openvino_model_server-2023.1-main-pr-image-mirror
/pj-rehearse pull-ci-opendatahub-io-openvino_model_server-2023.1-release-images
/pj-rehearse pull-ci-opendatahub-io-openvino_model_server-2023.1-main-nightly-images
/pj-rehearse pull-ci-opendatahub-io-openvino_model_server-2023.1-release-nightly-images

**Comment by rpancham** [comment]
/pj-rehearse
/pj-rehearse pull-ci-opendatahub-io-openvino_model_server-2023.1-main-pr-image-mirror
/pj-rehearse pull-ci-opendatahub-io-openvino_model_server-2023.1-release-images
/pj-rehearse pull-ci-opendatahub-io-openvino_model_server-2023.1-main-nightly-images
/pj-rehearse pull-ci-opendatahub-io-openvino_model_server-2023.1-release-nightly-images

**Comment by rpancham** [comment]
/pj-rehearse pull-ci-opendatahub-io-openvino_model_server-main-images

**Review by Xaenalt — APPROVED** [review]

**Pull request merged**